frontend/src/utils/locale.js

```diff
@@ -24,3 +24,13 @@ export function setCurrentActiveLocale(lang) {
   Moment.locale(lang);
   axios.defaults.headers.common['Accept-Language'] = lang;
 }
+
+/**
+ * @method isGermanLanguage
+ * @summary Returns boolean value if the language is german or not
+ */
+export function isGermanLanguage(languageObj) {
+  return languageObj && languageObj.key
+    ? languageObj.key.includes('de')
+    : false;
```

Comment:
why not using the `isGermanLanguage` into Table.js since u took the code from there?

![image](https://user-images.githubusercontent.com/1244701/112647083-d1018e00-8e50-11eb-911a-a2329a41728d.png)

Reply:
<img width="1263" alt="Screenshot 2021-03-27 at 07 06 18" src="https://user-images.githubusercontent.com/1708561/112710598-2a0e0800-8ecb-11eb-848b-893ac4f3ce40.png">
But that is exactly what I did. You are looking somewhere in your branch and not in the actual PR